src/panels/beaconSummaryPanel/BeaconSummaryEditing.tsx

```diff
@@ -153,6 +123,43 @@ export const BeaconSummaryEditing: FunctionComponent<{
                       }
                     />
 
+                    <TabulatedRow
+                      displayKey={
+                        <Typography>
+                          {"Protocol code" + WritingStyle.KeyValueSeparator}
+                        </Typography>
+                      }
+                      value={<FieldValue>{beacon.protocolCode}</FieldValue>}
+                    />
+
+                    <TabulatedRow
+                      displayKey={
+                        <Typography>
+                          {"Coding method" + WritingStyle.KeyValueSeparator}
+                        </Typography>
+                      }
+                      value={<FieldValue>{beacon.codingMethod}</FieldValue>}
+                    />
+
+                    <TabulatedRow
+                      displayKey={<></>}
+                      value={
+                        <Paper style={{ backgroundColor: "#FFFCC8" }}>
+                          <Box p={1}>
+                            <Typography>
+                              <b>Protocol code</b>, <b>coding method</b> and{" "}
+                              <b>beacon type</b> are automatically derived from
+                              the HEX ID. If you have identified a coding issue,
+                              please flag this record as 'Incorrectly Encoded'
+                              by clicking on the 'Quick Actions' button in the
+                              top right. You can also add any notes in the
+                              'Notes' tab below.
+                            </Typography>
+                          </Box>
+                        </Paper>
```

Comment:
Option in future to extract this to a reusable `<InfoText>` component if this pattern sticks.

![image](https://user-images.githubusercontent.com/54271433/118648482-5b73c780-b7da-11eb-9797-05f0eb995022.png)